Tasks/GooglePlayReleaseV4/googleutil.ts

```diff
@@ -0,0 +1,274 @@
+    return new googleapis.Auth.JWT(key.client_email, null, key.private_key, GOOGLE_PLAY_SCOPES, null);
+}
+
+/**
+ * Uses the provided JWT client to request a new edit from the Play store and attach the edit id to all requests made this session
+ * Assumes authorized
+ * @param packageName - unique android package name (com.android.etc)
+ * @return edit - A promise that will return result from inserting a new edit
+ *                          { id: string, expiryTimeSeconds: string }
+ */
+export async function getNewEdit(edits: pub3.Resource$Edits, packageName: string): Promise<pub3.Schema$AppEdit> {
+    tl.debug('Creating a new edit');
+    const requestParameters: pub3.Params$Resource$Edits$Insert = {
+        packageName: packageName
+    };
+
+    tl.debug('Additional Parameters: ' + JSON.stringify(requestParameters));
+    const res = await edits.insert(requestParameters);
+    return res.data;
+}
+
+/**
+ * Gets information for the specified app and track
+ * Assumes authorized
+ * @param packageName - unique android package name (com.android.etc)
+ * @param track - one of the values {"internal", "alpha", "beta", "production"}
+ * @returns track - A promise that will return result from updating a track
+ *                            { track: string, versionCodes: [integer], userFraction: double }
+ */
+export async function getTrack(edits: pub3.Resource$Edits, packageName: string, track: string): Promise<pub3.Schema$Track> {
+    tl.debug('Getting Track information');
+    const requestParameters: pub3.Params$Resource$Edits$Tracks$Get = {
+        packageName: packageName,
+        track: track
+    };
+
+    tl.debug('Additional Parameters: ' + JSON.stringify(requestParameters));
+    const getTrack = await edits.tracks.get(requestParameters);
+    return getTrack.data;
+}
+
+/**
+ * @param edits Google API Edits
+ * @param packageName unique android package name (com.android.etc)
+ * @param track release track. Should be one of {"internal", "alpha", "beta", "production"}
+ * @param versionCodes version codes that will be exposed to the users of this track when this release is rolled out
+ * @param userFraction for rollouting out a release to a track, it's the fraction of users to get update; 1.0 is all users
+ * @param updatePriority in-app update priority value of the release. All newly added APKs in the release will be considered at this priority. Can take values in the range [0, 5], with 5 the highest priority. Defaults to 0.
+ * @param releaseNotes optional release notes to be attached as part of the update
+ * @param releaseName optional release name. If not set, the name is generated from the APK's version_name. If the release contains multiple APKs, the name is generated from the date
+ * @returns track - A promise that will return result from updating a track
+ *                            { track: string, versionCodes: [integer], userFraction: double }
+ */
+export async function updateTrack(
```

Comment:
As I can there is already an `async function updateTrack` function in the  `GooglePlay.ts` file
https://github.com/microsoft/google-play-vsts-extension/blob/4522cf9e8f65ec59795cfa1bb4384803dc0d3fe1/Tasks/GooglePlayReleaseV4/GooglePlay.ts#L311-L322

Could you please change the name of the function? Also, it will be great to describe the difference between functions.

Reply:
Renamed in https://github.com/microsoft/google-play-vsts-extension/pull/292/commits/08ccf8f24c16b9238061d58d4530ae88777ec6ed. Not sure if there is a good name for it.
Also going to describe the difference between these functions later. Maybe a better name will come up in the process.

Reply:
Reworked in https://github.com/microsoft/google-play-vsts-extension/pull/292/commits/79b751997799937003df25ca7612984ef23f11db